lib/promenade/setup.rb

```diff
@@ -1,16 +1,25 @@
 require "pathname"
 
 module Promenade
-  def self.root_dir
-    rails_root = defined?(Rails) && Rails.root
-    rails_root || Pathname.new(ENV.fetch("RAILS_ROOT", Dir.pwd))
+  module_function
+
+  def root_dir
+    if rails_defined?
+      Rails.root
+    else
+      Pathname.new(ENV.fetch("RAILS_ROOT", Dir.pwd))
+    end
+  end
+
+  def rails_defined?
+    defined?(Rails)
```

Comment:
Added this so I could stub the behaviour for the setup specs. they were failing otherwise, because Rails is being loaded in other specs now